.vsts-ci/linux.yml

```diff
@@ -80,23 +80,24 @@ jobs:
   - template: ../tools/releaseBuild/azureDevOps/templates/insert-nuget-config-azfeed.yml
 
   - powershell: |
-      tools/travis.ps1 -Stage Bootstrap
+      Import-Module .\tools\ci.psm1
+      Invoke-LinuxTests
     displayName: Bootstrap
     condition: succeeded()
 
   - powershell: |
       $ErrorActionPreference = 'continue'
-      tools/travis.ps1 -NugetKey $(NUGET_KEY)
+      tools/ci.psm1 -NugetKey $(NUGET_KEY)
```

Comment:
This doesn't look right. You probably wanted to invoke a function that's _inside_ the psm1.

Reply:
```suggestion
      Import-Module tools/ci.psm1
      Invoke-AppveyorFinish -NugetKey $(NUGET_KEY)
```

Reply:
It isn't letting me accept the suggestion